mirrord-operator/templates/config-map.yaml

```diff
@@ -18,4 +27,5 @@ data:
     {{- end }}
     {{- if .Values.agent.extraConfig }}
     {{- .Values.agent.extraConfig | toYaml | nindent 4 }}
-    {{- end }}
\ No newline at end of file
+    {{- end }}
```

Comment:
need to move this `end` to line 21, since now you did that only if `Values.agent` is defined then imagePullSecrets will be defined..

Reply:
moved